### PR TITLE
feat(context): add BindQuery func

### PR DESCRIPTION
**Pull request by easonlin404**
see https://github.com/gin-gonic/gin/issues/742#issuecomment-268944445 , only parse/bind the query string params

### CODE REVIEW

**Comment by appleboy** [comment]
@easonlin404 Please also add a new section on readme.

**Comment by easonlin404** [comment]
@appleboy Already update readme. Thanks.

File: README.md [hunk 1]
```diff
@@ -454,7 +454,40 @@ func main() {
...
-### Bind Query String
+### Only Bind Query String
+
+`BindQuery` function only binds the query params and not the post data. See the [detail information](https://github.com/gin-gonic/gin/issues/742#issuecomment-315953017).
+
+```go
+package main
+
+import "log"
+import "github.com/gin-gonic/gin"
```
Comment (appleboy) [inline README.md]:
```go
import (
	"log"

	"github.com/gin-gonic/gin"
)
```

Comment (easonlin404) [inline README.md]: fixed. thanks for your kind reminder.

**Pull request merged**